packages/editor-ui/src/stores/workflows.ts

```diff
@@ -227,6 +227,10 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 				this.nodeMetadata[nodeName] && this.nodeMetadata[nodeName].parametersLastUpdatedAt;
 		},
 
+		getIsNodePristine(): (name: string) => boolean {
```

Comment:
nitpick - can simply be `isNodePristine`... two verbs is confusing

Reply:
I agree that the name is a bit clumsy but the `get` keyword IMO better invokes that this getter accepts parameter

Reply:
then maybe `getNodePristine`..  though personally prefer `is` for boolean states..

Reply:
I also like `is` more for the boolean state. But this isn't one "yet", it's a function that accepts the param and gets the boolean state. But I'm not married to this name :D and TS should indicate that it's a function anyway.

Reply:
ya but we call it like a function `isNodePristine(nodeName)`